src/Microsoft.Data/PrimitiveDataFrameColumnContainer.BinaryOperations.cs

```diff
@@ -0,0 +1,162 @@
+﻿
```

Comment:
Nit, newline

Reply:
Same as the prev comment. I really wish I knew where in the ttinclude file this was coming from